submissions/adeyemiJuwon/HTMLforms/account.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
```

Comment:
Well done! Update this file to be more descriptive like `signup.html`

Reply:
okay